src/pages/DirectMessage/apis/getMessageGroupListAPI.ts

```diff
@@ -0,0 +1,13 @@
+import { AUTH_API } from "@/apis/Api"
+
+const getMessageGroupListAPI = async () => {
+  // 인증을 매번 해야하는 번거로움 존재
+  return await AUTH_API.get("messages/conversations")
+    .then((res) => res.data)
+    .catch((e) => {
+      console.log(e, "전체 메시지 목록 받아옴 실패")
```

Comment:
[P3] 정말 사소한데 error는 console.error로 보여주는 게 더 좋을 것 같아요!🤔
```suggestion
     console.error(e, "전체 메시지 목록 받아옴 실패")
```